### PR TITLE
`prefers-color-scheme` article for web.dev

### PR DESCRIPTION
**Pull request by tomayac**
This pull request adds a **new**:
- [x] Guide(s) 📘
- [ ] Codelab(s) 🔬

Have you read the wiki on writing [a guide](https://github.com/GoogleChrome/web.dev/wiki/How-to-write-a-post) or [a codelab](https://github.com/GoogleChrome/web.dev/wiki/How-to-write-a-codelab)?
- [x] Done! 🍰(but adhering is WIP) 

Does your content tell a user how to fix a Lighthouse audit?
Have you added [the Lighthouse audit ID to your YAML](https://github.com/GoogleChrome/web.dev/wiki/How-to-write-a-guide#lighthouse-audits)?
- [x] My changes do not mention Lighthouse audits. 🙅‍♂️
- [ ] I have added the Lighthouse audit IDs to my content. 👍

Does your content reference any Glitch projects?
Have you read the wiki on [working with Glitch](https://github.com/GoogleChrome/web.dev/wiki/Working-with-Glitch)?
Have you added the Glitch project to [our team page](https://glitch.com/@webdev)?
- [x] My changes do not contain any Glitches. 🙅‍♂️
- [ ] I have read the wiki on working with Glitch. 🍰
- [ ] I have added my Glitches to the team page. 👏

### CODE REVIEW

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+title: Hello Darkness, My Old Friend—Dark Theme is Here
+subhead:
+authors:
+  - thomassteiner
+date: 2019-05-23
+hero: macosx.png
+alt: macOS X Dark Mode preferences.
+description: |
```
Comment (robdodson) [inline src/site/content/en/blog/prefers-color-scheme/index.md]: need a description so it'll appear in search.

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+tags:
```
Comment (robdodson) [inline src/site/content/en/blog/prefers-color-scheme/index.md]:
```suggestion
tags:
  - post
```

post tag is required to show up on the blog landing page. I need to figure out a better way to make sure all blog posts get it automatically :\

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
+---
+title: Hello Darkness, My Old Friend—Dark Theme is Here
+subhead:
```
Comment (robdodson) [inline src/site/content/en/blog/prefers-color-scheme/index.md]: Need a catchy subhead. This will appear on the post's card on the blog landing page and under the title on the article itself.

**Comment by jpmedley** [comment]
I'm told beta is expected at 10:00 am, PDT tomorrow. Will I have a link for the beta post by then?

**Comment by tomayac** [comment]
@jpmedley Sorry, I won’t make it until then. Can you rephrase to something “we’ll have a dedicated article on this soon”?

**Comment by robdodson** [comment]
Oh I didn't realize this post was intended for the beta blog. I thought it was just a test related to the capabilities migration. Really sorry if I held this up.

**Comment by tomayac** [comment]
@robdodson No, absolutely not. I just haven’t added all the things I wanted to. There’s quite some code examples that should go along with the post, all WIP.

**Comment by jpmedley** [comment]
Rob,

If you'll add a beta-related label to the repo, I'll start using it.

Joe
Joe Medley | Technical Writer, Chrome DevRel | jmedley@google.com |
 816-678-7195
*If an API's not documented it doesn't exist.*

**Comment by carolrivero** [comment]
+1 to beta labels

**Comment by robdodson** [comment]
sure thing. Added a beta blog label

**Comment by tomayac** [comment]
Sorry, I realized somehow this PR contains completely unrelated changes. Let me reopen.

**Comment by tomayac** [comment]
☝️ Started fresh in https://github.com/GoogleChrome/web.dev/pull/1021.
